### ISSUE TITLE
documentation for `eval_f` is not complete

### ISSUE THREAD

**Issue by arilamstein** [BAAQMD/funtools#2]
When I run R CMD check I get these issues:
> ── R CMD check results ──────────────────────────────── funtools 2021.05.27 ────
Duration: 22.4s

> checking Rd \usage sections ... WARNING
  Undocumented arguments in documentation object 'eval_f'
    ‘x’ ‘...’
  
>  Functions with \usage entries need to have the appropriate \alias
  entries, and all their arguments documented.
  The \usage entries must correspond to syntactically valid R code.
  See chapter ‘Writing R documentation files’ in the ‘Writing R
  Extensions’ manual.

> checking Rd contents ... WARNING
  Argument items with no description in Rd object 'eval_f':
    ‘env’

> checking Rd files ... NOTE
  prepare_Rd: eval_f.Rd:12-14: Dropping empty section \value
  
The actual code for `eval_f` can be found [here](https://github.com/BAAQMD/funtools/blob/master/R/eval_f.R). I reproduce it below:
```
#' eval_f
#'
#' @param x
#' @param ...
#' @param env
#'
#' @importFrom rlang as_quosure eval_tidy
#'
#' @return
#' @export
eval_f <- function (x, ..., env = NULL) {
  quosured <- rlang::as_quosure(x, env = env)
  rlang::eval_tidy(quosured, ...)
}
```
In short, the parameters are _named_ but not _described_ in the documentation. 

I don't have enough experience with rlang to properly describe these variables. But it would be nice if someone updated the documentation so the options are described.
